app/src/main/java/org/fossasia/openevent/app/data/attendee/AttendeeApi.java

```diff
@@ -10,7 +10,7 @@
 
 public interface AttendeeApi {
 
-    @GET("events/{id}/attendees?include=order,ticket,event&fields[event]=id&fields[ticket]=id&page[size]=0")
```

Comment:
Why this?

Reply:
It was just for debugging, changed it back now.